lib/components/Actions/OneDropdownButton/OneDropdownButton.tsx

```diff
@@ -36,7 +36,7 @@ const OneDropdownButton = ({
   value,
   ...props
 }: OneDropdownButtonProps) => {
-  const [localValue, setLocalValue] = useState(value || items[0].value)
+  const [localValue] = useState(value || items[0].value)
```

Comment:
do we need the local value?

Reply:
i can be a memo instead of a state as it will not change